error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/AssertJRules.java

```diff
@@ -311,7 +311,7 @@ static final class AssertThatMultisetsAreEqual<S, T extends S> {
   // `assertThat` overload. Consider defining a `BugChecker` instead.
   static final class AssertThatMapContainsEntry<K, V> {
     @BeforeTemplate
-    ObjectAssert<?> before(Map<K, V> map, K key, V value) {
+    ObjectAssert<V> before(Map<K, V> map, K key, V value) {
```

Comment:
I'd noticed this one didn't match on our end; wasn't sure if this change factored in, but didn't see a reason to have a wildcard here. Also wasn't sure why this test isn't in `AssertJMapRules`, as it seemed to better fit in there.

Reply:
The change is fine :+1:. This rule hasn't been moved to `AssertJMapRules` yet because in practice it will almost never match, for the reason outlined in the `XXX` comment.